aqua/resources-api.aqua

```diff
@@ -7,6 +7,22 @@ import "@fluencelabs/aqua-lib/builtin.aqua"
 alias ResourceId: string
 alias Error: string
 
+func wait(successful: *bool, n: i16) -> bool:
```

Comment:
What is `n`? Is it number of elements?

I suggest to rename `n` to `length`, and instead of 
```
join successful[n]
```
do
```
join successful[n - 1]
```